services/navigation/builtin/builtin.go

```diff
@@ -358,6 +373,7 @@ func (svc *builtIn) Close(ctx context.Context) error {
 }
 
 func (svc *builtIn) startWaypoint(ctx context.Context, extra map[string]interface{}) {
+	svc.logger.Debug("startWaypoint called")
```

Comment:
can we change the name of this function?  It doesn't describe what it does at all really

Reply:
done, what do you think of `startWaypointMode`?